docs/configuration/configuration-file.mdx

```diff
@@ -71,158 +62,416 @@ resolve:
+```
 
-registry:
-# The API registry options
+**INSERT SCHEMA.YAML HERE**
 
-# future products may be configured by extending this file
-```
+## `organization`
 
+The `organization` section stores your Redocly Workflows organization ID that the `push` command uses to push API definition files to the API registry. If an organization ID is not passed explicitly in command-line arguments, `push` will look for it in `redocly.yaml`.
 
-### organization
+The `organization` property is only required in the configuration file if you're using the API registry and/or hosted Reference docs. It is not required for local Redocly CLI usage.
 
-The `organization` property requires the organization ID of your Redocly Workflows organization.
-When specified in the configuration file, it can be used by the `openapi push` command.
-The `push` command uses the following order of precedence: first, it takes the organization ID from command-line arguments (if provided).
-If the organization ID is not provided explicitly, it takes it from the configuration file.
+To find the organization ID:
 
-Note that the organization ID is not the same as the organization name in Workflows.
+1. Log into Workflows.
+2. Access the **API registry** page.
+3. In your browser's address bar, find the page URL.
+4. The segment after `app.redocly.com/org/` is the organization ID.
 
-To find the organization ID, log into Workflows and access the **API registry** page.
-In your browser's address bar, find the URL of the page.
-The segment after `app.redocly.com/org/` is your organization ID.
 For example, if the URL is `app.redocly.com/org/test_docs`, the organization ID is `test_docs`.
 
+```yaml
+organization: test_docs
+apis:
+  main@v1:
+    root: ./openapi/openapi.yaml
+...
+```
 
-### apis
+:::info
+In Redocly Workflows, organization ID and organization name are not the same thing.
+:::
+
+## `apis`
+
+### Overview
 
-The `apis` section is used to configure one or more APIs. Every API in the section is identified by its name and version in the format `name@version`. The version is optional, and when not provided, Redocly apps interpret it as `latest` by default. Every `name@version` combination listed in the section must be unique.
+The `apis` section lets you configure one or more API definition files. This gives you the flexibility to reference specific files in commands, and configure each file at a granular level.
 
-For every API listed in the section, you must provide the path to the OpenAPI definition using the `root` property.
+Every file listed in this section is identified by `name@version`. `@version` is optional. It allows you to manage multiple versions of each API. When a version is not provided, Redocly apps interpret a file as being `latest`. Every `name@version` combination must be unique.
+
+For every OpenAPI definition listed in the `apis` section, use the `root` property to define the path to the file. For example:
```

Comment:
note: I didn't copy it because this is incorrect. This section is about APIs and not files. Let me try to explain by example. Same file, but two different APIs.

```yaml
apis:
  core@v1:
    root: ./openapi.yaml
  core@v2:
    root: ./openapi.yaml
    lint: {...}
```